pyeasee/charger.py

```diff
@@ -398,10 +398,10 @@ async def update_firmware(self):
         except (ServerFailureException):
             return None
 
-    async def set_dynamic_charger_circuit_current(self, currentP1: int, currentP2: int = None, currentP3: int = None):
+    async def set_dynamic_charger_circuit_current(self, currentP1: int, currentP2: int = None, currentP3: int = None, timeToLive: int = None):
```

Comment:
This should be timeToLive = 0 not timeToLive = None, if 0 = behave as before, right?